.github/workflows/csprng_randomness_tests.yml

```diff
@@ -0,0 +1,85 @@
+name: CSPRNG randomness testing Workflow
+
+env:
+  CARGO_TERM_COLOR: always
+  ACTION_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+  RUSTFLAGS: "-C target-cpu=native"
+  RUST_BACKTRACE: "full"
+
+on:
+  # Allows you to run this workflow manually from the Actions tab as an alternative.
+  workflow_dispatch:
+
+
+jobs:
+  setup-ec2:
+    name: Setup EC2 instance (csprng-randomness-tests)
+    if: ${{ github.event_name == 'pull_request_review' && github.event.review.state == 'approved' }}
+    runs-on: ubuntu-latest
+    outputs:
+      runner-name: ${{ steps.start-instance.outputs.label }}
+      instance-id: ${{ steps.start-instance.outputs.ec2-instance-id }}
+    steps:
+      - name: Start instance
+        id: start-instance
+        uses: zama-ai/slab-github-runner@ab65ad70bb9f9e9251e4915ea5612bcad23cd9b1
+        with:
+          mode: start
+          github-token: ${{ secrets.SLAB_ACTION_TOKEN }}
+          slab-url: ${{ secrets.SLAB_BASE_URL }}
+          job-secret: ${{ secrets.JOB_SECRET }}
+          profile: cpu-small
+
+  csprng-randomness-tests:
+    name: CSPRNG randomness tests
+    if: ${{ github.event_name == 'pull_request_review' && github.event.review.state == 'approved' }}
```

Comment:
this probably is not needed ? as we have a need on setup-ec2 ? I don't recall

Reply:
So it seems that the if isn't correctly set since none of the tests triggered upon approval were launched.

Reply:
alright debug time, I think the if should not be required except for the first job

Reply:
Ok let's try again. We have to start a series of approvals to validate my changes. Hopefully two of them should be sufficient.

Reply:
btw @soonum can we launch tests manually if needed on a PR ?

Reply:
You can do a workflow dispatch, I don't know if it appears in the check suite though.

Reply:
No it doesn't appear.